Users/abehenderson/CSX/Recursion/repeater.js

```diff
@@ -0,0 +1,31 @@
+// Attempt #2 (CSX Solution)
+
+function repeater(char) {
+  if (char.length === 5) return char;
+    return repeater(char + char[0]);
```

Comment:
When `repeater` is called here, the argument `char`'s value is, in the first run, `gg`, in the second run `ggg` ....in the fourth run `ggggg` where it is returned (backwards down the call stack).

Reply:
Ok so with every stack frame push, `char` is modified, and the final result `ggggg`is returned (because the breaking condition has been met) back to each previous stack frame down the call stack until it is returned to global memory?

Reply:
yes